SG Agreement.md

```diff
@@ -43,6 +43,11 @@ its interests, cast votes, and otherwise represent the interests of the [Steerin
 [Steering Group Member] may replace its [Steering Group Representative] at any time in its sole
 discretion.
 
+Each [Steering Group Member] may optionally appoint an individual
+"<a id="alternate-steering-group-representative">**Alternate Steering Group Representative**</a>". The
+[Steering Group Representative] may delegate any of their rights and responsibilities to the
+[Alternate Steering Group Representative] (except the ability to designate an alternate).
```

Comment:
You wrote that the SG Member has the ability to appoint the SG Alternate Representative, not the SG Representative. I think that's probably the way to go, but that does mean there's no need for "except".

Reply:
Removed.